lax/lichens/sciencerun0.py

```diff
@@ -206,7 +206,7 @@ class S2Width(ManyLichen):
 
```

Comment:
Replace the Wiki note linked above here, and update the description with what you've done.